### PR TITLE
Added typing for react-column-resizer

### PR DESCRIPTION
**Pull request by ibrahim-13**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

> Test failed with the following error, which is most probably an issue related to `dtslint` installation rather then the type definition

```
Error: Errors in typescript@4.2 for external dependencies:
../react/index.d.ts(38,22): error TS2307: Cannot find module 'csstype' or its corresponding type declarations.

    at D:\Projects\DefinitelyTyped\node_modules\dtslint\bin\index.js:206:19
    at Generator.next (<anonymous>)
    at fulfilled (D:\Projects\DefinitelyTyped\node_modules\dtslint\bin\index.js:6:58)
npm ERR! Test failed.  See above for more details.
```

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain `{ "extends": "dtslint/dt.json" }`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson), and no additional rules.
- [x] `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/react-column-resizer/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+import { Component } from 'react';
+
+interface ResizerProps {
+    className?: string;
+    disabled?: boolean;
+    minWidth?: number;
+}
+
+export default class ColumnResizer extends Component<ResizerProps> {}
+export {};
```
Comment (peterblazejewicz) [inline types/react-column-resizer/index.d.ts:15]:
```suggestion
```

File: types/react-column-resizer/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+// Type definitions for react-column-resizer 1.1
+// Project: https://github.com/nik-m2/react-column-resizer#readme
+// Definitions by: ibrahim <https://github.com/ibrahim-13>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { Component } from 'react';
+
+interface ResizerProps {
```
Comment (peterblazejewicz) [inline types/react-column-resizer/index.d.ts:8]:
```suggestion
iexport nterface ResizerProps {
```
it will make `export {}` redundant

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@ibrahim-13 thx!

**Comment by ibrahim-13** [comment]
@peterblazejewicz thanks to you too, I did learn something new from your code review!

**Comment by orta** [comment]
nice, very simple and straight forwards 👍🏻

**Pull request merged**